video/out/vo_dmabuf_wayland.c

```diff
@@ -390,7 +390,6 @@ static void destroy_osd_buffers(struct vo *vo)
 
     // Remove any existing buffer before we destroy them.
     wl_surface_attach(vo->wl->osd_surface, NULL, 0, 0);
```

Comment:
I'm not sure if this can also be removed, i.e., if the code that eventually flows into `flip_page` would update the old_surface buffer as well.

Reply:
This was needed because it used to cause glitches with stats.lua (Shift+I) overlays on wlroots, but I don't see any even without this anymore.

Reply:
Weston would segfault without this. Having checked yet if the ordering fixes that too.

Reply:
Yeah this code can be removed too. The only time it doesn't go to flip_page is during uninit which does not matter.

Reply:
Done